pkgs/by-name/Xr-Fragments/package.nix

```
@@ -0,0 +1,46 @@
+{
+  stdenv,
+  fetchgit,
+  haxe,
+  nodejs,
+  yarn,
+  makeWrapper,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
```

Replace [rec](https://nix.dev/guides/best-practices#recursive-attribute-set-rec) with finalAttrs (which is better for [overriding](https://wiki.nixos.org/wiki/Overlays)), whenever possible:

```suggestion
stdenv.mkDerivation (finalAttrs: {
```